Index: lib/DJabberd/RosterStorage/SQLite.pm
===================================================================
--- lib/DJabberd/RosterStorage/SQLite.pm
+++ lib/DJabberd/RosterStorage/SQLite.pm	2008-09-26 11:54:55.000000000 +0200
@@ -22,6 +22,7 @@
     die "No 'Database' configured'" unless $self->{dbfile};
 
     my $dbh = DBI->connect_cached("dbi:SQLite:dbname=$self->{dbfile}","","", { RaiseError => 1, PrintError => 0, AutoCommit => 1 });
+    $dbh->{unicode} = 1;
     $self->{dbh} = $dbh;
     $self->check_install_schema;
     return $self;
